.pep8speaks.yml

```diff
@@ -0,0 +1,11 @@
+scanner:
+  diff_only: True  # Errors caused by only the patch are shown, not the whole file
+
+pycodestyle:
+  max-line-length: 80  # Default is 79 in PEP8
```

Comment:
Some editors wrap onto exactly 80, so it's a bit annoying to be nitpicky about this.  I feel pretty ambivalent about 79 or 80—it must just display properly on the terminal.